python/lancedb/remote/client.py

```diff
@@ -76,38 +76,38 @@ def headers(self) -> Dict[str, str]:
         return headers
 
     @staticmethod
-    async def _check_status(resp: aiohttp.ClientResponse):
-        if resp.status == 404:
-            raise LanceDBClientError(f"Not found: {await resp.text()}")
-        elif 400 <= resp.status < 500:
+    def _check_status(resp: requests.Response):
+        if resp.status_code == 404:
+            raise LanceDBClientError(f"Not found: {resp.text}")
+        elif 400 <= resp.status_code < 500:
             raise LanceDBClientError(
-                f"Bad Request: {resp.status}, error: {await resp.text()}"
+                f"Bad Request: {resp.status_code}, error: {resp.text}"
             )
-        elif 500 <= resp.status < 600:
+        elif 500 <= resp.status_code < 600:
             raise LanceDBClientError(
-                f"Internal Server Error: {resp.status}, error: {await resp.text()}"
+                f"Internal Server Error: {resp.status_code}, error: {resp.text}"
             )
-        elif resp.status != 200:
+        elif resp.status_code != 200:
             raise LanceDBClientError(
-                f"Unknown Error: {resp.status}, error: {await resp.text()}"
+                f"Unknown Error: {resp.status_code}, error: {resp.text}"
             )
 
     @_check_not_closed
-    async def get(self, uri: str, params: Union[Dict[str, Any], BaseModel] = None):
+    def get(self, uri: str, params: Union[Dict[str, Any], BaseModel] = None):
         """Send a GET request and returns the deserialized response payload."""
         if isinstance(params, BaseModel):
             params: Dict[str, Any] = params.dict(exclude_none=True)
-        async with self.session.get(
-            uri,
+        with self.session.get(
+            self.url + uri,
             params=params,
             headers=self.headers,
-            timeout=aiohttp.ClientTimeout(total=30),
+            timeout=30,
```

Comment:
nit: maybe use `(5.0, 30.0)`. 5s for connection timeout and 30s for read timeout